components/link/index.css

```diff
@@ -10,55 +10,95 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-link.css";
+.spectrum-Link {
+  /* Colors */
+  /* Primary */
+  --spectrum-link-text-color-primary-default: var(--spectrum-accent-content-color);
+  --spectrum-link-text-color-primary-hover: var(--spectrum-accent-content-color-hover);
+  --spectrum-link-text-color-primary-active: var(--spectrum-accent-content-color-down);
+  --spectrum-link-text-color-primary-focus: var(--spectrum-accent-content-color-key-focus);
 
-.spectrum-Link--sizeS {
-  @remapvars {
-    find: /--spectrum-link(.*)-s-/;
-    filter: color;
-    replace: --spectrum-link$1-;
-  }
+  /* Secondary */
+  --spectrum-link-text-color-secondary-default: var(--spectrum-neutral-content-color-default);
+  --spectrum-link-text-color-secondary-hover: var(--spectrum-neutral-content-color-hover);
+  --spectrum-link-text-color-secondary-active: var(--spectrum-neutral-content-color-down);
+  --spectrum-link-text-color-secondary-focus: var(--spectrum-neutral-content-color-key-focus);
+
+  --spectrum-link-text-color-white: var(--spectrum-white);
+  --spectrum-link-text-color-black: var(--spectrum-black);
 }
 
-.spectrum-Link--sizeM {
-  @remapvars {
-    find: /--spectrum-link(.*)-m-/;
-    filter: color;
-    replace: --spectrum-link$1-;
+/* Windows high contrast mode */
+@media (forced-colors: active) {
+  &--staticBlack {
+    --highcontrast-link-text-color-black: ButtonFace;
   }
-}
 
-.spectrum-Link--sizeL {
-  @remapvars {
-    find: /--spectrum-link(.*)-l-/;
-    filter: color;
-    replace: --spectrum-link$1-;
+  &--staticWhite {
+    --highcontrast-link-text-color-white: ButtonFace;
   }
-}
 
-.spectrum-Link--sizeXL {
-  @remapvars {
-    find: /--spectrum-link(.*)-xl-/;
-    filter: color;
-    replace: --spectrum-link$1-;
+  .spectrum-Link--secondary {
+    color: LinkText;
+
+    &:hover {
+      color: LinkText;
+    }
+
+    &:active {
+      color: LinkText;
+    }
+
+    &:focus {
+      color: LinkText;
+    }
   }
 }
 
 .spectrum-Link {
-  font-size: var(--spectrum-link-primary-text-size);
   /* Remove the gray background on active links in IE 10. */
   background-color: transparent;
   /* Remove gaps in links underline in iOS 8+ and Safari 8+. */
   -webkit-text-decoration-skip: objects;
   text-decoration: underline;
-  transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
+  transition: color var(--spectrum-animation-duration-100) ease-in-out;
```

Comment:
Should this be established as a custom property up top?

Reply:
That makes sense. Updated.